src/lib/fragments/Accordion.react.js

```diff
@@ -206,18 +206,14 @@ class Accordion extends Component {
               onKeyDown={(event) => this.handleKeyEvent(event, index)}
               ref={this.headerRefs[index]}
             >
-              <span className="govuk-accordion__section-heading-text" >
-                <span className="govuk-accordion__section-heading-text-focus"> {accordionHeading}
-                </span>
-              </span>
               <span className="govuk-visually-hidden govuk-accordion__section-heading-divider"></span>
               <span className="govuk-accordion__section-toggle" data-nosnippet>
                 <span className="govuk-accordion__section-toggle-focus">
                   <span className="moj-side-navigation__item--collapsed">
                   </span>
                 </span>
                 <span className={sectionIsOpen ? "govuk-accordion-nav__chevron" : "govuk-accordion-nav__chevron govuk-accordion-nav__chevron--down"} ></span>
-                <span className="govuk-accordion__section-toggle-focus govuk-accordion__section-toggle-text"> {sectionIsOpen ? "Hide" : "Show"}
+                <span className="govuk-accordion__section-toggle-focus govuk-accordion__section-toggle-text"> {sectionIsOpen ? "Hide " + accordionHeading : "Show " + accordionHeading}
```

Comment:
```suggestion
                <span className="govuk-accordion__section-toggle-focus govuk-accordion__section-toggle-text"> {sectionIsOpen ? `Hide ${accordionHeading}` : `Show ${accordionHeading}`}
```

```suggestion
                <span className="govuk-accordion__section-toggle-focus govuk-accordion__section-toggle-text"> { ` ${ sectionIsOpen ? 'Hide' ? : 'Show' } ${accordionHeading}` }
```


Sorry i hate string concat